blender_bevy_toolkit/definitions/bevy_pbr/directional_light.py

```diff
@@ -0,0 +1,173 @@
+        )
+
+    @staticmethod
+    def unregister():
+        bpy.utils.unregister_class(DirectionalLightPanel)
+        bpy.utils.unregister_class(DirectionalLightProperties)
+        del bpy.types.Object.bevy_directional_light_properties
+
+
+class DirectionalLightPanel(bpy.types.Panel):
+    bl_idname = "OBJECT_PT_directional_light_properties"
+    bl_label = "BevyDirectionalLight"
+    bl_space_type = "PROPERTIES"
+    bl_region_type = "WINDOW"
+    bl_context = "physics"
+
+    @classmethod
+    def poll(cls, context):
+        return DirectionalLight.is_present(context.object)
+
+    @staticmethod
+    def draw(self, context):
+        row = self.layout.row()
+        row.label(text="Provider of directional illumination")
+
+        row = self.layout.row()
+        row.prop(context.object.data, "color")
+        row = self.layout.row()
+        row.prop(context.object.data, "energy", text="Illuminance")
+        row = self.layout.row()
+        row.prop(context.object.data, "use_shadow", text="Enable Shadow")
+
+        shadow_enabled = context.object.data.use_shadow
+        row = self.layout.row()
+        row.active = shadow_enabled
+        row.prop(context.object.data, "shadow_buffer_bias")  # Bevy Depth Bias
+        row = self.layout.row()
+        row.active = shadow_enabled
+        row.prop(context.object.bevy_directional_light_properties, "shadow_normal_bias")
+
+        box = self.layout.box()
+        box.active = shadow_enabled
+        box.label(text="Shadow Projection")
+        row = box.row()
+        row.prop(context.object.bevy_directional_light_properties, "left")
+        row = box.row()
+        row.prop(context.object.bevy_directional_light_properties, "right")
+        row = box.row()
+        row.prop(context.object.bevy_directional_light_properties, "bottom")
+        row = box.row()
+        row.prop(context.object.bevy_directional_light_properties, "top")
+        row = box.row()
+        row.prop(context.object.bevy_directional_light_properties, "near")
+        row = box.row()
+        row.prop(context.object.bevy_directional_light_properties, "far")
+        row = box.row()
+        row.prop(context.object.bevy_directional_light_properties, "scale")
+
+
+class DirectionalLightProperties(bpy.types.PropertyGroup):
+    shadow_normal_bias: bpy.props.FloatProperty(name="Shadow Normal Bias", default=0.0)
+
+    # Shadow Orthographic Projection properties
+    left: bpy.props.FloatProperty(name="left", default=-100.0)
+    right: bpy.props.FloatProperty(name="right", default=100.0)
+    bottom: bpy.props.FloatProperty(name="bottom", default=-100.0)
+    top: bpy.props.FloatProperty(name="top", default=100.0)
+    near: bpy.props.FloatProperty(name="near", default=-100.0)
+    far: bpy.props.FloatProperty(name="far", default=100.0)
+    scale: bpy.props.FloatProperty(name="scale", default=1.0)
+    # Missing window_origin, scaling_mode, depth_calculation
+
+
+register_component(
```

Comment:
These two components are the same as the ones for the PointLight, so I think it would be super cool to break them into a separate file and change the `is_present_function` to be `lambda obj: DirectionalLight.is_present(obj) or PointLight.is_present(obj)`

I also suspect that blender may be spitting out a warning about trying to register the same component twice because these components are duplicated.

Reply:
Actually, I take that back! They are not the same as the ones in PointLight! My bad,